app/models/apple/publisher.rb

```diff
@@ -106,20 +106,30 @@ def poll!(eps = episodes_to_sync)
       end
     end
 
-    def publish!(eps = episodes_to_sync)
+    def publish!
       show.sync!
       raise "Missing Show!" unless show.apple_id.present?
 
-      # delete or unpublished episodes
+      # Archive deleted or unpublished episodes.
+      # These episodes are no longer in the private feed.
       poll_episodes!(episodes_to_archive)
       archive!(episodes_to_archive)
-      show.reload
```

Comment:
These calls are only needed when episodes move from `apple_new? == true` to `apple_new? == false`. Or when an episode moves in and out of a private feed (which should not be happening during a publishing flow)